spec/migrator/alter_extension_statement_spec.cr

```diff
@@ -0,0 +1,13 @@
+require "../spec_helper"
+
+describe Avram::Migrator::AlterExtensionStatement do
+  it "builds the proper SQL to update an extension" do
+    statement = Avram::Migrator::AlterExtensionStatement.new("uuid-ossp")
+    statement.build.should eq %{ALTER EXTENSION "uuid-ossp" UPDATE;}
+  end
+
+  it "updates to a specific version" do
+    statement = Avram::Migrator::AlterExtensionStatement.new("hstore", to: "2.0")
+    statement.build.should eq %{ALTER EXTENSION "hstore" UPDATE TO '2.0';}
```

Comment:
I didn’t know altering extensions was a thing! Nice that you added it

Reply:
There's actually a TON of altering you can do, but just specifying a specific version seemed to be the most common use I found. We can always extend this later if people start needing more options.

Reply:
Sounds perfect to me! People can always drop down to `execute` if they need to 👍

Reply:
Exactly. We have escape hatches for it, so this should cover the 90% 😄